usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -7,6 +7,12 @@
     get_jvm_logger,
     create_ref_temp_views,
 )
+from usaspending_api.recipient.delta_models.recipient_profile import (
```

Comment:
Not worth blocking over, but can you follow pattern of alphabetical ordering?

Reply:
Re-ordered import statements